src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -190,6 +190,9 @@ internal EmitBaseline GetDelta(Compilation compilation, Guid encId, MetadataSize
             // otherwise members from the current compilation have already been merged into the baseline.
             var synthesizedMembers = (_previousGeneration.Ordinal == 0) ? module.GetAllSynthesizedMembers() : _previousGeneration.SynthesizedMembers;
 
+            Debug.Assert(module.EncSymbolChanges is not null);
+            var deletedMembers = (_previousGeneration.Ordinal == 0) ? module.EncSymbolChanges.GetAllDeletedMethods() : _previousGeneration.DeletedMembers;
```

Comment:
I didn't do this originally because it is not needed, as it is used when re-adding deleted members, and its impossible to do that in generation 1, but without it the `DeletedMembers` property of the EmitBaseline doesn't represent the truth for generation 1, which was annoying for tests.